lametro/templates/partials/meeting_details_next.html

```diff
@@ -32,6 +32,7 @@ <h4>
                         {% else %}
                           <br>
                           <p class="small"><em>Agenda will be posted no later than 72 hours prior to the start of the meeting, or 24 hours for Special Board meetings.</em></p>
+                          <p><strong>No meetings are in session at this time, The links to watch a meeting live, in English or Spanish, will be posted here once the meeting begins.</strong></p>
```

Comment:
also: let's display this message, regardless of whether there are agenda documents.